src/birdie/internal/titles.gleam

```diff
@@ -238,29 +238,32 @@ fn snap_call(
     glance.Call(
       function:,
       arguments: [
-        glance.Field(None, title),
-        glance.Field(Some("content"), _snapshot_content),
+        glance.UnlabelledField(title),
+        glance.LabelledField("content", _snapshot_content),
       ],
     )
     | glance.Call(
         function:,
         arguments: [
-          glance.Field(None, _snapshot_content),
-          glance.Field(_, title),
+          glance.UnlabelledField(_snapshot_content),
+          glance.LabelledField(_, title),
         ],
       )
     | glance.Call(
         function:,
         arguments: [
-          glance.Field(Some("content"), _snapshot_content),
-          glance.Field(_, title),
+          glance.LabelledField("content", _snapshot_content),
+          glance.LabelledField(_, title),
```

Comment:
Same as above, this is no longer matching the same thing

Reply:
This now also matches one where the second argument is unlabelled